hono/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "wrangler dev",
     "migrate": "tsx migrate.ts",
     "generate": "drizzle-kit generate:pg",
-    "test": "jest --silent --detectOpenHandles --testPathIgnorePatterns='neon.preflight' --testPathIgnorePatterns='remote.preflight' --testPathIgnorePatterns='/preflight'",
+    "test": "jest --detectOpenHandles --testPathIgnorePatterns='neon.preflight' --testPathIgnorePatterns='remote.preflight' --testPathIgnorePatterns='/preflight'",
     "test:smoke": "jest --silent --testPathPattern=smoke.test.ts$",
```

Comment:
Might be a good idea to drop `--silent` from all test scripts to help with debugging

Reply:
Done.